addon/initializers/route-layers.js

```diff
@@ -1,6 +1,8 @@
+import Ember from 'ember';
 import RouteLayers from 'ember-route-layers/services/route-layers';
 
-export function initialize (container, application) {
+export function initialize () {
+  let application = arguments[1] || arguments[0];
```

Comment:
It looks like we thought this would be fixed with the latest release, but I was still seeing it in Ember 2.4. I did it this way from [the docs](http://emberjs.com/deprecations/v2.x/#toc_initializer-arity).